spec/datasets/djornl/edge_type.yaml

```diff
@@ -23,3 +23,32 @@ oneOf:
   - const: AraNetv2-LC_lit-curated-ppi
     title: AraNetv2 literature-curated protein-protein interaction
     description: A layer of protein-protein interaction values from literature-curated small- to medium-scale experimental data; from the Arabidopsis AraNetv2 database. The LLS scores that serve as edge values were calculated to normalize the data for comparison across studies and different types of data layers (Lee et al, 2015).
+
+  - const: BIOGRID-ORGANISM-Arabidopsis-thaliana-Columbia-3-5-188-tab3_PPI
+    title: BIOGRID ORGANISM Arabidopsis thaliana Columbia 3.5.188 tab3 PPI
+    description: BioGRID interactions are relationships between two proteins or genes; the term “interaction” includes direct physical binding of two proteins, and co-existence in a stable complex and genetic interaction.  see https://wiki.thebiogrid.org/doku.php/experimental_systems
+
+  - const: AtPIN_PPI
+    title: AtPIN PPI
+    description: The interactions database includes all interactions present at the Arabidopsis thaliana Protein Interactome Database, the Predicted Interactome for Arabidopsis, Arabidopsis protein-protein interaction data curated from the literature by TAIR curators, BIOGRID and IntAct. https://atpin.bioinfoguy.net/cgi-bin/atpin.pl
+      
+  - const: mentha-AT-3702-040319_PPI
+    title: Mentha AT 3702 040319 PPI
+    description: Mentha archives evidence collected from different sources and presents these data in a complete and comprehensive way. Its data comes from manually curated protein-protein interaction databases that have adhered to the IMEx consortium and assigns to each interaction a reliability score that takes into account all the supporting evidence.  https://mentha.uniroma2.it/about.php
+
+  - const: aranetv2-subnet-AT-LC_PPI
+    title: AraNetv2 subnet AT-LC PPI
+    description: Literature curated PPI of A. thaliana. https://www.inetbio.org/aranet/dl.php?f=AT-LC
+
+  - const: Wu2016-s015-Gene-to-Metab_GeneToPhenotype
+    title: Wu2016 s015 Gene-to-Metab GeneToPhenotype
+    description: GWAS hits ftom a Gene-to-Metaboiltes GWAS. Phenotypes (metabolites) have a unique ID from the Wu 2016 study and need to be given our own UID for future use.
+
+  - const: AraGWAS-subnet-permsig-geni_GeneToPhenotype
```

Comment:
Is this different to the existing network `AraGWAS-Phenotype_Associations`? What does "permsig geni" mean?

Reply:
I believe this is a sub-network of `AraGWAS-Phenotype_Associations`. The notes I have for the file say ...

```Has the best GWAS hits from AraGWAS (i.e. passed a permutation-based threshold). 17,606 edges```

whereas I think the full `AraGWAS-Phenotype_associations` network is on the order of 1,000,000 edges.